src/NewTwitchApi/NewTwitchApi.php

```diff
@@ -14,15 +14,47 @@
 
 class NewTwitchApi
 {
-    private $oauthApi;
-    private $gamesApi;
-    private $streamsApi;
-    private $usersApi;
-    private $webhooksApi;
-    private $webhooksSubscriptionApi;
-
-    public function __construct(Client $helixGuzzleClient, string $clientId, string $clientSecret, Client $authGuzzleClient = null)
+    /**
+     * @var OauthApi
+     */
+    protected $oauthApi;
+
+    /**
+     * @var GamesApi
+     */
+    protected $gamesApi;
+
+    /**
+     * @var StreamsApi
+     */
+    protected $streamsApi;
+
+    /**
+     * @var UsersApi
+     */
+    protected $usersApi;
+
+    /**
+     * @var WebhooksApi
+     */
+    protected $webhooksApi;
+
+    /**
+     * @var WebhooksSubscriptionApi
+     */
+    protected $webhooksSubscriptionApi;
+
+    /**
+     * NewTwitchApi constructor.
+     *
+     * @param string $clientId
+     * @param string $clientSecret
+     * @param Client|null $authGuzzleClient
+     */
+    public function __construct(string $clientId, string $clientSecret, Client $authGuzzleClient = null)
     {
+        $helixGuzzleClient = new HelixGuzzleClient($clientId);
```

Comment:
Why remove this from the constructor? This is hard-coding the class to this particular client. It is useful to be able to inject your own client with different settings (e.g. with added logging middleware, etc).